### DISCUSSION TITLE
Download from blob protocol. Ex.: blob:https....

### DISCUSSION THREAD

**Discussion by LuizAntonioJR-RJ** [sindresorhus/got#1757] [Q&A] [answered]
Im getting this message:

`"Unsupported protocol "blob:" "`

The link i need to download the PDF is build through a link like this: blob:https://DOMAIN/141d4184-3dec-403e-9a59-ea134c61c8c1.

Can anyone helps me out, please?

**Comment by sindresorhus** [comment, marked as answer]
`Blob` is not available on Node.js. Also, `blob:` is a browser-internal URL. There's probably a better way to solve your problem, but it's not related to Got. I would recommend asking on Stack Overflow instead.

https://stackoverflow.com/questions/30864573/what-is-a-blob-url-and-why-it-is-used

**Reply by LuizAntonioJR-RJ** [reply]
Thanks a lot my friend. I didnt know that.
